Tests/PSGetInstallModule.Tests.ps1

```diff
@@ -23,6 +23,16 @@ function SuiteSetup {
     $script:PSGetLocalAppDataPath = Get-PSGetLocalAppDataPath
     $script:TempPath = Get-TempPath
     $null = New-Item -Path $script:MyDocumentsModulesPath -ItemType Directory -ErrorAction SilentlyContinue -WarningAction SilentlyContinue
+    
```

Comment:
Is that a best practice?  I thought I had to put all Update-Module tests in the PSGetUpdateModule.Tests.ps1 file, and similarly with the other cmdlets?
If this is alright with you, this will make my life a LOT easier :)

---
In reply to: [132599444](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132599444) [](ancestors = 132599444)